inc/downloads.inc

```diff
@@ -670,13 +657,7 @@ else
 	fi
     fi
   
-if [[ -f /usr/bin/xz && "$INITIALINSTALL" = [yY] ]] && [[ "$PHPXZ_CHECK" != '5.5' || "$PHPXZ_CHECK" != '5.6' || "$PHPXZ_CHECK" != '5.7' || "$PHPXZ_CHECK" != '7.0' || "$PHPXZ_CHECK" != '7.1' ]]; then
   tar $PHPTAR_FLAGS "php-${PHP_VERSION}.tar.${PHPEXTSION}"
-elif [ -f /usr/bin/xz ]; then
-  XZ_OPT='--memory=max' tar $PHPTAR_FLAGS "php-${PHP_VERSION}.tar.${PHPEXTSION}"
-else
-  tar $PHPTAR_FLAGS "php-${PHP_VERSION}.tar.${PHPEXTSION}"
-fi
```

Comment:
Unnecessary option XZ_OPT.
Look at [(http://git.tukaani.org/?p=xz.git;a=blob;f=NEWS;hb=HEAD)]
5.0.0 (2010-10-23)
"The memory usage limit is now disabled by default. Some scripts written before this change may
have used --memory=max on xz command line or in XZ_OPT. THESE USES OF --memory=max SHOULD BE
REMOVED NOW, because they interfere with user's ability to set the memory usage limit himself."